next-tavla/src/Shared/styles/global.css

```diff
@@ -47,12 +52,6 @@ h3 {
     justify-content: space-between;
 }
 
-.flexRow {
-    display: flex;
-    flex-direction: row;
-    gap: 1rem;
-}
-
```

Comment:
Har du rebaset på master? dette er vel en endring som har blitt utført for "lenge" siden og ikke i denne branchen?

Reply:
Nei jeg la til den 2. oktober, så la du til den mer generelle 5., også fjernet jeg denne nå etter rebase